mysql-test/suite/innodb/t/bootstrap.test

```diff
@@ -35,7 +35,7 @@ EOF
 write_file $bugdir/data/mysql/plugin.frm;
```

Comment:
What is the current path of this file in 5.7 HEAD?

Reply:
In 5.7 it used to be `mysql-test/suite/innodb/t/log_file.test`

Reply:
The same `sync_latch_meta_init()` memory leak. Possibly others as well.